translations/list.yml

```diff
@@ -1,16 +1,16 @@
-description: 'Lists all available commands'
-help: "The <info>%command.name%</info> command lists all commands:\n<info>php %command.full_name%</info>\nYou can also display the commands for a specific namespace:\n<info>php %command.full_name% test</info>\nYou can also output the information in other formats by using the <comment>--format</comment> option:\n<info>php %command.full_name% --format=xml</info>\nIt's also possible to get raw list of commands (useful for embedding command runner):\n<info>php %command.full_name% --raw</info>\n"
+description: 'सर्व उपलब्ध आदेशांची यादी करा.'
+help: "<info>% कमांड.नाव% </info> आदेश सर्व कमांड्सची सूची देते: \n <info> php% आदेश.पूर्णता @ </info> \n आपण विशिष्ट नेमस्पेश साठी आदेश देखील दर्शवू शकता: \n <info> Php% आदेश.पूर्णता_नाम% test </info> \n आपण <comment> - format </comment> पर्यायाचा वापर करून इतर स्वरुपात माहिती काढू शकता: \n <info> php% आदेश.पूर्णता_नाम% - रूपमा = Xml </info> \n आदेशांची कच्ची यादी मिळविणेही शक्य आहे (आदेश रननर एम्बेड करण्यासाठी उपयोगी): \n <info> php% आदेश.पूर्णता_नाम% --raw</info> \n"
```

Comment:
Dont change the contnet inside tags that are not translatable like <comment>--format</comment>